eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -491,6 +490,9 @@ class EclairImpl(appKit: Kit) extends Eclair with Logging {
   override def usableBalances()(implicit timeout: Timeout): Future[Iterable[UsableBalance]] =
     (appKit.relayer ? GetOutgoingChannels()).mapTo[OutgoingChannels].map(_.channels.map(_.toUsableBalance))
 
+  override def channelBalances()(implicit timeout: Timeout): Future[Iterable[UsableBalance]] =
```

Comment:
I think this should return something slightly different than `UsableBalance`: it would be useful to include the `isEnabled` boolean from the channel update. When the channel is disabled, this conceptually wouldn't be a `UsableBalance` anymore...

My suggestion is to rename `UsableBalance` to `ChannelBalance` and enrich it with a new field `isEnabled: Boolean`. The existing `usablebalances` API can then also return an `Iterable[ChannelBalance]`, where `isEnabled` will always be `true` (it's a bit redundant, but explicit, which is good).